projects/plugins/jetpack/sal/class.json-api-site-base.php

```diff
@@ -1458,4 +1470,22 @@ public function get_launchpad_checklist_tasks_statuses() {
 
 		return array();
 	}
+
+	/**
+	 * Get site option for the production blog id (if is a WP.com Staging Site).
+	 *
+	 * @return string
+	 */
+	public function get_wpcom_production_blog_id() {
+		return get_option( 'wpcom_production_blog_id', '' );
```

Comment:
@wojtekn Interestingly, the staging site doesn't return the production blog id...

<img width="1598" alt="image" src="https://user-images.githubusercontent.com/36432/221719551-b29b64e3-8556-4474-bf30-1dbc3e80f4cd.png">

Reply:
I added D103003-code to fix that.

Reply:
Tested and now it's working 👌